extide/gradle/src/org/netbeans/modules/gradle/ActionProviderImpl.java

```diff
@@ -149,7 +149,7 @@ public void invokeAction(String command, Lookup context) throws IllegalArgumentE
                 prjImpl.primeProject().
                         thenAccept(gp -> {
                             LOG.log(Level.FINER, "Priming build of {0} finished with status {1}, ", new Object[] { project, prjImpl.isProjectPrimingRequired() });
-                            prg.finished(prjImpl.isProjectPrimingRequired());
+                            prg.finished(!prjImpl.isProjectPrimingRequired());
```

Comment:
@sdedic, would you agree this condition should be reverted?

Reply:
oh, absolutely; thanks.